app/templates/buyers/create_brief_question.html

```diff
@@ -11,20 +11,21 @@
 %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
+  {{ govukBreadcrumbs({
+    "items": [
       {
-        "link": "/",
-        "label": "Digital Marketplace"
+        "text": "Digital Marketplace",
+        "href": "/"
       },
       {
-        "link": url_for("dos.info_page_for_starting_a_brief", framework_slug=framework.slug, lot_slug=lot.slug),
-        "label": previous_title_text
+        "text": previous_title_text,
+        "href": url_for("dos.info_page_for_starting_a_brief", framework_slug=framework.slug, lot_slug=lot.slug)
+      },
+      {
+        "text": question.question
```

Comment:
This seems wrong?

![image](https://user-images.githubusercontent.com/503614/72897658-5ef85b00-3d1a-11ea-9d33-389da2861c25.png)

Reply:
Changed to parent template's heading `question.question`.